http-ts/README.md

```diff
@@ -0,0 +1,47 @@
+# TypeDB HTTP Typescript Driver
+
+## Driver Architecture
+
+To learn about how the TypeDB HTTP driver communicates with the TypeDB Server,
+refer to the [HTTP API Reference](https://typedb.com/docs/reference/http-api).
```

Comment:
Note: this link is going to die soon, but Ill make a task to update all links in our readmes